admin/tools/docker/deployment/in2p3/runQueries.py

```diff
@@ -124,12 +128,17 @@
     declMin = random.uniform(-87, 40)
     raDist = random.uniform(8, 12)
     declDist = random.uniform(8, 12)
-#    QUERY_POOLS["nearN"].append("select o1.ra as ra1, o2.ra as ra2, o1.decl as decl1, o2.decl as decl2, scisql_angSep(o1.ra, o1.decl,o2.ra, o2.decl) AS theDistance from Object o1, Object o2 where qserv_areaspec_box(%f, %f, %f, %f) and scisql_angSep(o1.ra, o1.decl, o2.ra, o2.decl) < 0.015" % (raMin, declMin, raMin+raDist, declMin+declDist))
-    QUERY_POOLS["nearN"].append("select count(*) from Object o1, Object o2 where qserv_areaspec_box(%f, %f, %f, %f) and scisql_angSep(o1.ra, o1.decl, o2.ra, o2.decl) < 0.015" % (raMin, declMin, raMin+raDist, declMin+declDist))
+# QUERY_POOLS["nearN"].append("select o1.ra as ra1, o2.ra as ra2, o1.decl
```

Comment:
Even if this has been commented out for now it's format should be the same as the lower one which hasn't.